R/compute_scale.R

```diff
@@ -0,0 +1,118 @@
+#' Compute scale parameters
```

Comment:
```suggestion
#' Compute Scale Parameters
```

Title should be in Proper Case